pkg/core/native/native_gas.go

```diff
@@ -40,9 +41,9 @@ func NewGAS() *GAS {
 
 	g.nep5TokenNative = *nep5
 
-	onp := g.Methods["onPersist"]
-	onp.Func = getOnPersistWrapper(g.onPersist)
-	g.Methods["onPersist"] = onp
+	desc := &manifest.Method{Name: "onPersist", ReturnType: smartcontract.VoidType}
+	md := newMethodAndPrice(getOnPersistWrapper(g.onPersist), 0, smartcontract.AllowModifyStates)
+	g.AddMethod(md, desc, false)
```

Comment:
Not sure, but is it correct to use `g.AddMethod` here? We have this method called inside of `newNEP5Native(...)` during description of `onPersist` manifest method, so there's already one `onPersist` in Manifest.ABI.Methods.

Reply:
Missed that, fixed.